src/plopp/plotting/inspector.py

```diff
@@ -70,31 +79,27 @@ def inspector(
     :
         A :class:`Box` which will contain two :class:`Figure` and one slider widget.
     """
-    if obj.ndim != 3:
-        raise ValueError(
-            'The inspector plot currently only work with '
-            f'three-dimensional data, found {obj.ndim} dims.'
-        )
     require_interactive_backend('inspector')
 
-    da = preprocess(obj, ignore_size=True)
-    in_node = Node(da)
+    in_node = Node(preprocess, obj, ignore_size=True)
```

Comment:
What if `obj` is a node?

Reply:
it just stays a Node if it is already a Node: https://github.com/scipp/plopp/blob/main/src/plopp/core/node.py#L70